src/components/Layout/index.jsx

```diff
@@ -5,15 +5,18 @@ import { Helmet } from "react-helmet";
 export default function Layout({ children, title, location }) {
   return (
     <>
-      <Helmet>
-        <meta charSet="utf-8" />
-        <title>{`Bearcabulary ${title ? " • " + title : ""}`}</title>
-        <meta name="description" content="Word game for children K-3." />
-        <link
-          rel="canonical"
-          href={`https://bearcabulary.com/${location.pathname}`}
-        />
-      </Helmet>
+      <Helmet
+        htmlAttribute={{lang: 'en'}}
```

Comment:
You're definitely more knowledgeable than me for React Helmets so I'll approve it. I didn't know you can use them without <> tags.

Reply:
You definitely can't. I have them there.